tools/ocean_data_view_manager/tool_odv.xml

```diff
@@ -0,0 +1,189 @@
+<tool id="tool_odv" name="ODV collection manager" version="@VERSION@+galaxy@VERSION_SUFFIX@" profile="22.05" license="MIT">
+    <description>tool for qualificiation and validation</description>
+    <macros>
+        <token name="@VERSION@">0.2</token>
+        <token name="@VERSION_SUFFIX@">0</token>
+    </macros>
+    <requirements>
+        <container type="docker">pokapok/qcv-odv-tool:@VERSION@</container>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        export HOME=\$PWD &&
+        mkdir -p /runtime/config &&
+        mkdir -p '/runtime/data-in-raw/' &&
+        mkdir -p '/runtime/data-in-ref/' &&
+        mkdir -p '/runtime/data-out/' &&
+        mkdir -p ./outputs &&
+
+        cp '$input_raw' '/runtime/data-in-raw/${input_raw.element_identifier}' &&
+        #if $input_ref:            
+            cp '$input_ref' '/runtime/data-in-ref/${input_ref.element_identifier}' &&
+        #end if
+
+        
+        #if $config.type == 'yes':
+            ## Write YAML content
+            cp '${config_file}' '/runtime/config/config_user_odv_tool.yml' &&
+        #end if 
+
+        /app/launchers/start-app.sh '$mode' &&
+
+        #if $mode == 'R_APP_QV':
+            cp '/runtime/data-out/odv_collection.txt' '$output' &&
+        #else:
+            cp '/runtime/data-out/demo_odv_collection.txt' '$output' &&
```

Comment:
those file will also be in the collection correct? So they are doubled? Is that indendet?

Reply:
Yes correct ! 
The real output is the file.
But after discussion with the people that wants this tool they told me that some user may want to have a folder with everything also.
This took as to be tested out afterwards see if this folder is really needed or not.